eipw-lint/src/lints/markdown/relative_links.rs

```diff
@@ -25,13 +25,15 @@ impl Lint for RelativeLinks {
     fn lint<'a, 'b>(&self, slug: &'a str, ctx: &Context<'a, 'b>) -> Result<(), Error> {
         let re = Regex::new("(^/)|(://)").unwrap();
 
+        let cs_re = Regex::new(r"^https://(www\.)?github\.com/ethereum/consensus-specs(/|$)").unwrap();
```

Comment:
We have the additional requirement that links to consensus specs include a specific commit, so this should only match URLs like:

```
https://github.com/ethereum/consensus-specs/blob/26695a9fdb747ecbe4f0bb9812fedbc402e5e18c/...
```

Reply:
I wasn't aware, sorry about that. I will add it, just let me verify - it should point to a version of a file at some commit, therefore it has to be `../blob/..` or the commit itself, which would be  `.../commit/..`? 

Are there any other rules to follow? For example prefix, currently it allows all combinations with `https://` or `www` or without

Reply:
I should've mentioned earlier! It's entirely my bad.

I suppose the best option would be `https` only, allow both with and without the `www`.

Links should go to specific files at a specific commit, so I think `.../blob/...` is what we want.